hack/e2e-suite/certs.sh

```diff
@@ -1,51 +0,0 @@
-#!/bin/bash
```

Comment:
Same comment I had for @jlowdermilk : we should leave the shell test in for ~1 week to verify that the replacement go test is at least as good.